{
    "config": {
        "error": {
            "cannot_connect": "Ha fallado la conexi\u00f3n",
            "invalid_api_key": "Clave API inv\u00e1lida"
        },
        "step": {
            "user": {
                "data": {
                    "api_key": "Clave API",
                    "name": "Nombre",
                    "path": "Ruta",
                    "url": "URL"
                }
            }
        }
    }
}